### PR TITLE
Clean up the __init__

### PR DESCRIPTION
**Pull request by varunagrawal**
Simple PR to do some wrapper clean up.

### CODE REVIEW

File: python/gtsam/__init__.py [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
-from .gtsam import *
-from .utils import findExampleDataFile
+"""Module definition file for GTSAM"""
+
+# pylint: disable=import-outside-toplevel, global-variable-not-assigned, possibly-unused-variable, import-error, import-self
+
+import sys
+
+from gtsam import gtsam, utils
+from gtsam.gtsam import *
```
Comment (johnwlambert) [inline python/gtsam/__init__.py:8]: thanks for the PR @varunagrawal. While we're updating this, we probably want to remove this wildcard import too.

[Wildcard imports discussed in PEP8](https://www.python.org/dev/peps/pep-0008/)

File: python/CMakeLists.txt [hunk 2]
```diff
@@ -65,8 +65,10 @@ set(interface_headers
     ${PROJECT_SOURCE_DIR}/gtsam/basis/basis.i
 )
 
+set(GTSAM_TARGET gtsam_py)
+set(GTSAM_UNSTABLE_TARGET gtsam_unstable_py)
 
-pybind_wrap(gtsam_py # target
+pybind_wrap(${GTSAM_TARGET} # target
```
Comment (ProfFan) [inline python/CMakeLists.txt:71]: GTSAM_PYTHON_TARGET

File: python/CMakeLists.txt [hunk 3]
```diff
@@ -122,7 +124,7 @@ if(GTSAM_UNSTABLE_BUILD_PYTHON)
             gtsam::CameraSetCal3Fisheye
             gtsam::KeyPairDoubleMap)
             
-    pybind_wrap(gtsam_unstable_py # target
+    pybind_wrap(${GTSAM_UNSTABLE_TARGET} # target
```
Comment (ProfFan) [inline python/CMakeLists.txt:127]: same

File: python/gtsam/__init__.py (hunk 1 above)
Comment (ProfFan) [reply python/gtsam/__init__.py:8]: This is needed, and I think is generally OK for pybind11 modules.

**Pull request merged**